packages/kbn-babel-preset/node_preset.js

```diff
@@ -18,6 +18,25 @@
  */
 
 module.exports = (_, options = {}) => {
+  const overrides = [];
```

Comment:
There shouldn't actually be a need to remove them at all.
We're removing them just because we don't need them in prod and figured it would make people feel more comfortable about including them in TM for the perf test.

I can see the value in using a wrapper to keep TM's code a little neater... I'll revisit that after merging as I feel this branch has lived too long and merge conflicts are piling up.

Reply:
I agree we should find another solution for this - I really don't like the idea of modifying the build based on an environmental variable and this will need changed soon as we work towards hermetic builds.